docs/posts/roadmap-2024-H1/index.qmd

```diff
@@ -0,0 +1,219 @@
+---
+title: "Ibis project 2024 roadmap"
+author: "Cody Peterson"
+date: "2024-02-10"
+image: commits.png
+categories:
+    - blog
+    - roadmap
+    - community
+---
+
+## Overview
+
+Welcome to the first public roadmap for the Ibis project! If you aren't familiar
+with the background of Ibis or who supports it nowadays, we recommend reading
+[why Voltron Data supports Ibis](../why-voda-supports-ibis/index.qmd) before the
+roadmap below.
+
+## 2024 roadmap
```

Comment:
Besides the roadmap, do we also have a set of principles for prioritization/design decisions that we should keep in the public? I feel it's important for us to differentiate from an open-core approach to an open-standard approach.

Reply:
I don't think we have anything like this, but I'm also not sure we need it at the moment.

I also don't think positioning Ibis using the open core versus open standards terminology is the right approach and adds linguistic overhead and terminology that isn't particularly relevant for many users.

Reply:
I do agree this is something we should have (probably a concept article) but don't want to block on the roadmap for it -- it is relatively covered in the "why VoDa supports Ibis blog" that we don't intend to build an open-core SaaS around Ibis or anything

Reply:
ok, sg. The reason why I ask is when I spoke to an integrating partner yesterday, I sensed concerns or, at the minimum, a lack of information to understand Ibis' OSS motivation (vs backing company's commercial motivation). I think this blog post will help already; it's just something we'll have to continue to think about and to strenghen trust with the community. Ultimately, what works best is probably just focusing on providing value and exponential growth :)